ospo-mindmap/Content/ospomindmap.md

```diff
@@ -70,21 +84,80 @@ development team of the Linux Project
 
 - Security Standards
 - License Compliance Policies
-- Tooling
-  - Build in-house
-  - Adopt existing tools
-      - Commercial tools
-      - [Open Source tools](https://oss-compliance-tooling.org/Tooling-Landscape/OSS-Based-License-Compliance-Tools/)
-- Design OSS infrastructure
-
-### OS Community Education
 
-#### Create Internal OS Ambassadors
 
-- Drive event sponsorship
-- Train speakers for coding forums
-  
-#### Create educational programs
+### 📝 Establish and Improve Open Source Policies and Processess
+
+##### Policies
+
+- Using and compliance policy
+  - Forking an open source project
+  - Copying open source licensed files/snippets into organization's codebase
+  - Mixing code (owned and third-party OSS/proprietary) and distributing as products or services (sdk, REST API, SaaS, etc.)
+  - Using standalone open source tools (e.g. Firefox, MySQL, CentOS)
+- Policy on open source licenses
+- Policy on releasing code as open source
+  - Contributing back to third party OSS project
+  - Publishing a new open source project
+  - Contributing in personal time
+  - Signing contribution agreements (e.g. CLA or DCOs)
+- Policy on making open source business decisions
+  - Risk assessment and acceptance  
+  - Business alignment
+
+#### Processes
+
+- Creating Open Source
+  - Contributing to third-party OSS projects during work time
+    - Which copyright statement to use for contributions on behalf of the organization
+    - What to do if there is already a copyright notice in the code
+    - What to do if there is no copyright notice in the code
+    - What to do if the community object to any changes to the copyright notices
+  - Contributing to third-party OSS projects during personal time
+  - Publishing a new open source project
+    - Check if the new project name is already in use in the community or is trademarked
+  - Sponsoring Open Source
+    - Sponsoring third-party projects
+    - Sponsoring organization projects or individual members
+    - Sponsoring events
+  - Signing contribution related agreements
+    - Contributor license agreement
+    - Developer certificate of origin
+  - Shutting down a new open source project
+- Using Open Source
+  - Using an open source project as a standalone tool
+  - Including open source snippets or files
+  - Including open source packages
+  - Forking an open source project
+  - Inventorying open source usage incl. packages, licensing and communities
+- Deciding on Open Source
+  - Drive Open Source Review Board with senior leaders from relevant stakeholder to make decisions on open source
+  - Drive Open Source Working Group with representatives / interested stakeholders throughout the organization with the aim of improving open source at the organization
+- Marketing and/or communicating Open Source
+  - Internal messaging
```

Comment:
Under "Internal messaging" some additional nodes:

- Organize internal Open Source events
- Present Open Source at the organization's all-hands and similar meetings

Reply:
Or maybe these fall under "Establish and Improve Open Source Policies and Processess" --> "Processes" --> "Educating people"

Reply:
I updated Educating People section so it's now

  - Organizing events 
    - Present Open Source at the organization's all-hands and similar meetings
    - Organize brown bag seminars on open source topics (with invited speakers)